UI/allCandidate2.html

```diff
@@ -0,0 +1,164 @@
+						ALHAJI SULIAMON QUADRI
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span><br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+						
+					</div>
+					<div class="candidate-card-footer">
+						<small>The people Choice Party</small>
+					</div>
+				</div>
+				<div class="candidate-card">
+					<h2 class="candidate-card-title">
+						TAIWO OYINDAMOLA EZEKIEL
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span>
+					<br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+					</div>
+					<div class="candidate-card-footer">
+						<small>People Democratic Congress</small>
+					</div>
+				</div>
+				<div class="candidate-card">
+					<h2 class="candidate-card-title">
+						DR OLADIPO SHERIFF TOLUWANI
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span><br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+					</div>
+					<div class="candidate-card-footer">
+						<small>Action Congress Party</small>
+					</div>
+				</div>
+				<div class="candidate-card">
+					<h2 class="candidate-card-title">
+						ALHAJI SULIAMON QUADRI
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span><br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+					</div>
+					<div class="candidate-card-footer">
+						<small>The people Choice Party</small>
+					</div>
+				</div>
+				<div class="candidate-card">
+					<h2 class="candidate-card-title">
+						TAIWO OYINDAMOLA EZEKIEL
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span>
+					<br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+					</div>
+					<div class="candidate-card-footer">
+						<small>People Democratic Congress</small>
+					</div>
+				</div>
+				<div class="candidate-card">
+					<h2 class="candidate-card-title">
+						DR OLADIPO SHERIFF TOLUWANI
+					</h2>
+					<span class="candidate-card-profile">
+						Date of birth:  <b>15th, Aug 1985</b></span>
+					</span><br>
+					<div class="candidate-card-button-area">
+						<a href="">
+							<button>
+								Vote
+							</button>
+						</a>
+					</div>
+					<div class="candidate-card-footer">
+						<small>Action Congress Party</small>
+					</div>
+				</div>
+			</div>
+		</div>
+		<footer class="politico-footer">
+			<p>politico. &copy; 2019</p>
+		</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks